snappi/tests/test_getproperty.py

```diff
@@ -0,0 +1,12 @@
+import snappi
+
+
+def test_getproperty():
+    config = snappi.Api().config()
+    f = config.flows.flow()[-1]
+    s = f.getproperty('size')
+    assert s == None
+    fixed = f.getproperty('size', True).getproperty('fixed', True)
+    assert fixed == 64
+    r = f.getproperty('rate', True)
```

Comment:
please add a test for both get_default=False as well. moreover, test should ensure that object is not mutated when get_default=True.